torch/lib/c10d/test/ProcessGroupGlooTest.cpp

```diff
@@ -48,7 +48,7 @@ class SignalTest {
 
     // Initialize tensor list
     std::vector<at::Tensor> tensors = {
-        at::ones({16, 16}, at::TensorOptions(at::CPU(at::kFloat))),
+        at::ones({16, 16}, at::device(at::kCPU).dtype(at::kFloat)),
```

Comment:
The whole right hand side is unnecessary, CPU and float are default. CAn just be `at::ones({16, 16})`